CryptographicallyVerify.md

```diff
@@ -0,0 +1,76 @@
+<p align="left">
+  <img src="https://raw.githubusercontent.com/step-security/supply-chain-goat/main/images/Logo.png" alt="Step Security Logo" width="340">
+</p>
+
+# Tutorial: Cryptographically verify tools run as part of the CI/ CD pipeline
+
+## Summary of past incidents
+### SUNSPOT: An Implant in the Build Process
+In December 2020, the industry was rocked by the disclosure of a supply chain attack against SolarWinds, Inc. The supply chain attack was the result of inclusion of unauthorized malicious code during the build process. SUNSPOT is the name of the malware used to insert a backdoor into software builds of the SolarWinds Orion IT management product.
```

Comment:
Please add a link to the article. The tutorial on source code modification has this link.